Applications/AlgorithmPlaza.qmd

```diff
@@ -1,8 +1,9 @@
 ---
 title: "openEO algorithm plaza"
 ---
+The [openEO algorithm plaza](https://marketplace-portal.dataspace.copernicus.eu/){target="_blank"} will host many algorithms that are built on top of the openEO API. These can be addressed via user interfaces or via API to embed into new services or platforms. Third party services will be able to onboard their algorithms semi-automatically in the openEO plaza for further exposure. Users are granted with credits to explore the available algorithms and check if they are fit for purpose for their needs.
 
-This section of the document provides a comprehensive overview of essential links related to the openEO algorithm plaza. These links serve as valuable resources for individuals seeking to gain further knowledge about the platform. Included are references to a guide tailored for general users, a guide designed specifically for developers and service providers, as well as additional links pertaining to various services offered by openEO.
+In this section we provide a comprehensive overview of essential links related to the openEO algorithm plaza. These links serve as valuable resources for individuals seeking to gain further knowledge about the platform. Included are references to a guide tailored for general users, a guide designed specifically for developers and service providers, as well as additional links pertaining to various services offered by openEO.
```

Comment:
As a more general note, is it intentional that "openEO algorithm plaza" is written here and elsewhere in lower case? Being a service name, I would write it in title case "openEO Algorithm Plaza"

Reply:
updated